kolibri/core/assets/src/content_renderer_module.js

```diff
@@ -0,0 +1,13 @@
+const KolibriModule = require('kolibri_module');
+
+module.exports = class ContentRenderer extends KolibriModule {
```

Comment:
the class name `ContentRenderer` isn't used anywhere. Note it's possible to [omit the classname](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/class), which is useful here because it's immediately exported